client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -152,16 +160,41 @@ export default function WPCOMBusinessAT( { product }: Props ): ReactElement {
 
 	// Handles dispatching automated transfer.
 	const dispatch = useDispatch();
-	const initiateAT = React.useCallback( () => {
+	const initiateAT = useCallback( () => {
 		setShowDialog( false );
 		dispatch( initiateThemeTransfer( siteId, null, '' ) );
 	}, [ dispatch, siteId ] );
+
 	const eventName =
 		product === 'backup'
 			? 'calypso_jetpack_backup_business_at'
 			: 'calypso_jetpack_scan_business_at';
 	const trackInitiateAT = useTrackCallback( initiateAT, eventName );
 
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+
+	useEffect( () => {
+		// Detect when the transfer is complete to refresh the site data
+		if ( automatedTransferStatus === COMPLETE ) {
+			// Need to refresh the site data.
+			dispatch( requestSite( siteId ) );
+		}
+	}, [ automatedTransferStatus ] );
+
+	useEffect( () => {
+		if ( automatedTransferStatus === COMPLETE && isJetpack ) {
+			dispatch(
+				successNotice( translate( 'Jetpack features are now activated.' ), {
+					id: 'jetpack-features-on',
+					duration: 5000,
+					displayOnNextPage: true,
+				} )
+			);
+			// Reload the page, whatever siteSlug is
+			page( `/${ product }/${ siteSlug }` );
+		}
+	}, [ automatedTransferStatus, isJetpack ] );
```

Comment:
I wonder if we could combine these two effects into one?

Reply:
only with `isJetpack`?

Reply:
No, I mean something like this:
```js
useEffect( () => {
	if ( automatedTransferStatus !== complete ) exit;
	if ( ! isJetpack ) fetch site data and exit;
	redirect to correct url
}, [ automatedTransferStatus, isJetpack ] );
```